rllib/algorithms/dqn/dqn.py

```diff
@@ -497,6 +502,21 @@ def get_default_rl_module_spec(self) -> RLModuleSpec:
                 "Use `config.framework('torch')` instead."
             )
 
+    # TODO (sven, simon): We cannot use our `override` decorator here, as the
+    # this only works on an instance not a class. We should refactor this.
+    @property
+    def _model_auto_keys(self) -> Dict[str, Any]:
+        return super()._model_auto_keys | {
+            "double_q": self.double_q,
+            "dueling": self.dueling,
+            "epsilon": [(0, 1.0), (10000, 0.05)],
```

Comment:
Should this not be `"epsilon": self.epsilon,` here?

Reply:
Yes, very good catch @sven1977 !